src/main/resources/api.yaml

```diff
@@ -388,6 +388,12 @@ paths:
             format: int64
           required: true
           description: Numeric ID of the sprint
+        - in: query
```

Comment:
Tu w sumie chyba nie ma jednak po co robić osobnego zapytania na te trzy liczby, które chcemy wziąć z Jiry (skoro to i tak się ma uaktualnić w bazie po drodze)